cmd/vic-machine/configure/configure.go

```diff
@@ -109,6 +123,38 @@ func (c *Configure) processParams() error {
 // creation process, for example, image store path, volume store path, network slot id, etc. So we'll copy changes based on user input
 func (c *Configure) copyChangedConf(o *config.VirtualContainerHostConfigSpec, n *config.VirtualContainerHostConfigSpec) {
 	//TODO: copy changed data
+
+	if c.proxies.IsSet {
+		hProxy := ""
+		if c.HTTPProxy != nil {
+			hProxy = c.HTTPProxy.String()
+		}
+		sProxy := ""
+		if c.HTTPSProxy != nil {
+			sProxy = c.HTTPSProxy.String()
+		}
+		personaSession := o.ExecutorConfig.Sessions[config.PersonaService]
```

Comment:
Would it be better to initialize `personaSession` and `vicAdminSession` outside the `if` block in case they're needed by code for other reconfigure items?